public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuTrigger.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { FC, HTMLAttributes, MouseEvent, ReactElement, useCallback, useState } from 'react';
+
+interface ClickCoordinates {
+  x: number;
+  y: number;
+}
```

Comment:
We already have the same interface https://github.com/grafana/grafana/blob/1d689888b0fc2de2dbed6e606eee19561a3ef006/packages/grafana-ui/src/components/uPlot/plugins/ClickPlugin.tsx#L29 that's not exported from grafana ui. Maybe we can have this type in grafana/data like `CartesianCoords2D` so we can reuse it rather than recreating here and there? I bet we have more intarfaces like this.

Reply:
Yes, good suggestion!

Reply:
Fixed and changed ClickPlugin accordingly.